### DISCUSSION TITLE
typo in epub ...

### DISCUSSION THREAD

**Discussion by manes6969** [tabemann/zeptoforth#75] [General]
Hi tabemann ,
I think there's a small but important typo in the epub in the chapter i2c 

i2c-clock!

( clock i2c – )

Set the clock for an I2C peripheral. Note that for most purposes a clock of 4000000 is recommended, and it is this value to which the clock of an I2C peripheral defaults.

I think it should be 400000 instead of 4000000 .

Kind regards,
Ronny

**Comment by tabemann** [comment]
I have now corrected that typo in `docs/words/i2c.md` and regenerated the HTML and EPUB docs.

**Comment by manes6969** [comment]
Thanks !
